src/selectors/right-clicked-call-node-path.js

```diff
@@ -0,0 +1,38 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import { createSelector } from 'reselect';
+import { getCallNodeIndexFromPath } from '../profile-logic/profile-data';
+import { getThreadSelectors } from './per-thread';
+import { getProfileViewOptions } from './profile';
+
+import type {
+  CallNodePath,
+  IndexIntoCallNodeTable,
+} from '../types/profile-derived';
+import type { Selector } from '../types/store';
+
+export const getRightClickedCallNodePath: Selector<CallNodePath | null> = createSelector(
+  getProfileViewOptions,
+  viewOptions => {
+    return viewOptions.rightClickedCallNodePath
+      ? viewOptions.rightClickedCallNodePath.callNodePath
+      : null;
+  }
+);
```

Comment:
I think we need to return the full information CallNodePath +  ThreadIndex. Because when we'll handle the value we'll need to know what thread it relates to.